src/Components/Patient/PatientRegister.tsx

```diff
@@ -314,6 +318,12 @@ export const PatientRegister = (props: PatientRegisterProps) => {
           ? res.data.permanent_address
           : state.form.permanent_address,
       });
+      field.onChange({
+        name: "occupation",
+        value: res.data.meta_info?.occupation
+          ? res.data.meta_info.occupation
+          : state.form.occupation,
+      });
```

Comment:
The null check can be simplified

```suggestion
      field.onChange({
        name: "occupation",
        value: res.data.meta_info?.occupation ?? state.form.occupation,
      });
```